packages/app/src/utils/api.ts

```diff
@@ -0,0 +1,25 @@
+import jsCookie from "js-cookie";
```

Comment:
Why use a new "api" lib since `customProgrester` already does that?

Reply:
I'll try to mix up both

Reply:
Après avoir essayé, je n'y suis pas arrivé :( Le problème étant de récupérer la data par l'URL